MapzenSDK/MZMapViewController.swift

```diff
@@ -456,6 +457,7 @@ open class MZMapViewController: UIViewController, LocationManagerDelegate {
    - parameter seconds: How long the animation should last.
   */
   open func animate(toPosition position: TGGeoPoint, withDuration seconds: Float) {
+    if isInBackground { return }
```

Comment:
We would be fine without the `isInBackground` tests since we would let a 'pending' animation until the app is brought back from the background. Now, by adding those, we set this to be a no-op when the app is in the background. I think that it would be important to document that depending on what you end up choosing here (no-op vs pending).

Reply:
Agreed on documentation for the no-op. Not sure I follow the "pending animation" part? Do you mean on ES 0.9 ? This is still on 0.8 so allowing any calls through to Tangram-es will cause errors?

Reply:
Ah yes, this is still pending for 0.9.0 so don't mind it for now. You could consider changing this behavior when updating to the new version.